### DISCUSSION TITLE
Einmalige Speicherladung aktivieren

### DISCUSSION THREAD

**Discussion by m-ask** [john30/ebusd#747] [Q&A]
Hallo!
Mein Projekt mit der Vaillant geoTherm funktioniert mittlerweile sehr gut.
Eine Funktion die ich noch nicht hinbekommen habe ist die einmalige Speicherladung.
Ich sende entsprechend load on aber es funktioniert nichts. Wenn ich die Speicherladung über die Bedieneinheit aktiviere funktioniert sie ohne Probleme.
Hat jemand einen Tip, muss man evtl. den Wert zyklisch senden?
Danke
m-ask

**Comment by mwildbolz** [comment]
bei mir funktioniert es, indem ich den sfMode auf "load" stelle

**Reply by m-ask** [reply]
hallo mwildbolz
den sfMode gibt es nur in der 15.700?
bei mir wird die 15.uih.csv ausgwählt

**Comment by cyberthom42** [comment]
Kann man die Speicherladung auch wieder unterbrechen?

**Reply by mwildbolz** [reply]
ja, wenn man HwcOpMode auf OFF stellt

**Comment by michbeck3000** [comment]
Seit v24.1 kann ich bei meiner VWS 82/3 keine einmalige Speicherladung mehr aktivieren. In früheren Versionen war das kein Problem. Für das WW nimmt das System "vaillant/25.ehp.hwc.csv for ID "ehp00", SW0416, HW7201"
Kann mich jemand bitte auf den richtigen Pfad schubsen?

**Reply by martin8310** [reply]
Hallo, und hast Du das Problem gelöst?

**Reply by michbeck3000** [reply]
Ja, bei mir muss ich den Schalter select.ebusd_hwc_load_value auf on setzen, dann klappt es. Der setzt sich dann nach einer Weile selber wieder zurück.

**Comment by michbeck3000** [comment]
Hier noch meine Ausgabe von ebusctl info:
(config path: https://ebus.github.io)

2ad9b828-ebusd:/# ebusctl info
version: ebusd 24.1.p20241115
update check: revision 24.1 available, device firmware v5 1[5210] up to date
device: 192.168.178.97:9999, TCP, enhanced, firmware 1.1[5210].1[5210]
signal: acquired
symbol rate: 83
max symbol rate: 162
min arbitration micros: 1
max arbitration micros: 56
min symbol latency: 4
max symbol latency: 37
scan: finished
reconnects: 0
masters: 5
messages: 635
conditional: 219
poll: 152
update: 66
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=EHP00;SW=0416;HW=7201", loaded "vaillant/08.ehp.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=UIH00;SW=0370;HW=6901", loaded "vaillant/15.uih.csv"
address 23: slave, scanned "MF=Vaillant;ID=EHP00;SW=0416;HW=7201", loaded "vaillant/23.ehp.cc.csv"
address 25: slave, scanned "MF=Vaillant;ID=EHP00;SW=0416;HW=7201", loaded "vaillant/25.ehp.hwc.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 3f: master #23
address 50: slave, scanned "MF=Vaillant;ID=EHP00;SW=0416;HW=7201", loaded "vaillant/50.ehp.mc.csv"
address 7f: master #24
